src/library/scala/collection/StringOps.scala

```diff
@@ -28,17 +27,17 @@ final class StringOps(private val s: String)
   def toIterable = StringView(s)
   override def view: StringView = StringView(s)
   protected[this] def coll: String = s
-  override def toSeq: immutable.Seq[Char] = fromIterable(toIterable)
+  override def toSeq: immutable.Seq[Char] = s.toCharArray
```

Comment:
This relies on the deprecated implicit conversion that copies the array a 2nd time for the `ImmutableArray` after copying it from the String in `toCharArray`. Creating a `WrappedString` should be much more efficient.

Reply:
sorry! I didn't see this message in time, looks like you're on top of it.